src/EditorFeatures/Core/Implementation/SolutionPreviewResult.cs

```diff
@@ -18,6 +18,11 @@ internal class SolutionPreviewResult : ForegroundThreadAffinitizedObject
         private readonly IList<SolutionPreviewItem> _previews;
         public readonly SolutionChangeSummary ChangeSummary;
 
+        public SolutionPreviewResult(SolutionPreviewItem preview, SolutionChangeSummary changeSummary = null)
```

Comment:
Can you add doc comments to this class explaining that it supports preview merging (which appends previews but chooses the first provided summary)?